spec/slim_lint/linter/file_length_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe SlimLint::Linter::FileLength do
+  include_context 'linter'
+
+  context 'when a file contains too many lines' do
+    let(:slim) { (0..300).to_a.join("\n") }
+
+    it { should report_lint line: 1 }
+  end
+
+  context 'when a file does not contain lines which are too long' do
```

Comment:
`when a file does not contain lines which are too long` -> `when a file does not contain too many lines`